backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -511,22 +513,22 @@ def search_content(
         if modified_range:
             search = search.filter("range", modified=modified_range)
 
-        search.aggs.bucket("content_types", "terms", field="content_type.{}".format(KEYWORD_FIELD))
+        search.aggs.bucket("content_types", "terms", field="content_type")
         search.aggs.bucket(
-            "workspace_names", "terms", field="workspace.label.{}".format(KEYWORD_FIELD)
+            "workspace_names", "terms", field="workspace.label.{}".format(EXACT_FIELD)
```

Comment:
We finally decide to go for exact ?

Reply:
I didn't change this, the field was named like this in the model.

Reply:
However, I can get rid of either KEYWORD_FIELD or EXACT_FIELD, but this might annoy @grignards

Reply:
just stick with one name and it will be ok.

Reply:
Done in @grignards 's PR